app/controllers/home_controller.rb

```diff
@@ -14,6 +14,11 @@ def index
   def profile
   end
 
+  def reference_calculator
+    @iban = BankAccount.first.try(:iban)
+    @types = FinancialTransactionType.with_name_short.order('name ASC')
```

Comment:
this will need an explanation somewhere on the form where short name is entered and in the documentation, that only financial transaction types with a short name will be selectable in the reference calculator.